CONTRIBUTING.md

```diff
@@ -129,6 +129,9 @@ Before you submit your pull request consider the following guidelines:
     git rebase master -i
     git push origin my-fix-branch -f
     ```
+    
+
+WARNING. Forced push may delete all in-code comments made in your previous PR
```

Comment:
It's the same PR (you probably meant "previous commit").
I thought GitHub will retain the comments ? They just won't be shown in-line in your new commit.

I think this warning will be more confusing than helpful. We are advising people to force-push, but the warn them (as if it were a bad/dangrous thing).

--
I think we should either change the instructions to suggest creating a new commit with the changes (as I believe is suggested by @petebacondarwin) or not warn them against the inconveniences of force-pushing.

WDYT, @petebacondarwin ?